test/Header.test.js

```diff
@@ -252,7 +262,7 @@ describe('Header', () => {
           (done) => {
             patronApiCall(component, '/refreshError', deleteNyplIdentityPatronCookie);
             setTimeout(() => {
-              expect(deleteNyplIdentityPatronCookie.calledOnce).to.equal(true);
+              expect(deleteNyplIdentityPatronCookie.calledTwice).to.equal(true);
```

Comment:
I was trying to explain what happens here but then I realized there might a way for a better test condition. I will try make the updates.

Reply:
OK! I have updated the test so it will be called once now.

The reason why it was called twice is because after Mocha mounted <Header>, it would call the method to set Encore time, and since we didn't pass a valid last visited time for Encore page, it will set the timer to 0 and delete "nyplIdentityPatron". So this was the extra one.

I didn't think thorough enough before but now I realized I can just pass a mock value to "ENCORE_LAST_VISITED".